osbuild/formats/v2.py

```diff
@@ -493,3 +494,7 @@ def validate_pipeline(pipeline, path):
         validate_pipeline(pipeline, path=["pipelines", i])
 
     return result
+
+
+def format_osbuild_error(error: OSBuildError):
```

Comment:
I would like to comment on the commit message, but I'll put it here instead: Could you add some examples of the returned structs in the commit message to make it easier to picture for the reviewers (my brain's python interpreter can only do so much ;) )?

Reply:
I'll add this :)